sky/backends/backend_utils.py

```diff
@@ -1678,9 +1678,10 @@ def _query_status_gcp(
                      f'--filter="(labels.ray-cluster-name={cluster})" '
                      '--format="value(state)"')
     else:
+        # Ref: https://cloud.google.com/compute/docs/instances/instance-life-cycle
         status_map = {
             'PROVISIONING': global_user_state.ClusterStatus.INIT,
-            'STARTING': global_user_state.ClusterStatus.INIT,
+            'STAGING': global_user_state.ClusterStatus.INIT,
```

Comment:
Nice. I guess we never hit it otherwise we'd have a KeyError?

Reply:
I recently hit this during the smoke tests. I suppose the period of this state is pretty short, so we don't easily hit this state when doing `sky status -r`.